packages/documentation/src/stories/utilities/float/float.stories.ts

```diff
@@ -0,0 +1,82 @@
+import type { Args, StoryObj } from '@storybook/web-components';
+import { html } from 'lit/static-html.js';
+import { MetaExtended } from '@root/types';
+import './float.styles.scss';
+
+/* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+
+const meta: MetaExtended = {
+  id: 'ccf092c1-f0ab-49a1-a612-87e5be23adde',
+  title: 'Utilities/Float',
+};
+
+export default meta;
+
+type Story = StoryObj;
+
+export const Float: Story = {
+  argTypes: {
+    floatPosition: {
+      name: 'float',
+      description: 'Sets the float. ',
+      control: {
+        type: 'select',
+      },
+      options: ['start', 'end', 'none'],
+      table: {
+        category: 'Add float',
+      },
+    },
+    floatBreakPoint: {
+      name: 'float',
+      description: 'Sets the float for a specific breakpoint. ',
+      control: {
+        type: 'select',
+      },
+      options: ['', 'sm', 'md', 'lg', 'xl', 'xxl'],
+      table: {
+        category: 'Add float',
+      },
+    },
+  },
+  args: {
+    floatPosition: '',
+    floatBreakPoint: '',
+  },
+  render: (args: Args) => {
+    const float = args.floatPosition !== '' ? `float` : '';
+    const floatPosition = args.floatPosition !== '' ? `-${args.floatPosition}` : '';
+    const floatBreakPoint = args.floatBreakPoint !== '' ? `-${args.floatBreakPoint}` : '';
```

Comment:
```suggestion
    const classMembers = ['float', args.floatBreakPoint ?? 'xs', args.floatPosition].filter(m => m);
    const floatClass = classMembers.length === 3 ? classMembers.filter(m => m !== 'xs').join('-') : nothing;
```

This gathers the possible class parts (`float-{breakpoint}-{position}`) and passes them as a `-` separated string to `floatClass` if `args.floatPosition` is defined. The breakpoint `xs` will always be filtered out and if `args.floatPosition` is undefined, the `floatClass` will be defined as `nothing` (which is the lit way to set a `undefined` or `null` value).

Reply:
Great :)